doc/api.rst

```diff
@@ -204,47 +205,69 @@ Functions and utilities in the :mod:`sim` module, for simulating time series.
 Periodic Signals
 ~~~~~~~~~~~~~~~~
 
-.. currentmodule:: neurodsp.sim.periodic
+.. currentmodule:: neurodsp.sim
 
 .. autosummary::
-  :toctree: generated/
+    :toctree: generated/
 
-  sim_oscillation
-  sim_bursty_oscillation
+    sim_oscillation
+    sim_bursty_oscillation
 
 Aperiodic Signals
 ~~~~~~~~~~~~~~~~~
 
-.. currentmodule:: neurodsp.sim.aperiodic
+.. currentmodule:: neurodsp.sim
 
 .. autosummary::
-  :toctree: generated/
+    :toctree: generated/
 
-  sim_powerlaw
-  sim_poisson_pop
-  sim_synaptic_current
-  sim_random_walk
+    sim_powerlaw
+    sim_poisson_pop
+    sim_synaptic_current
+    sim_random_walk
 
 Transients
 ~~~~~~~~~~
 
-.. currentmodule:: neurodsp.sim.transients
+.. currentmodule:: neurodsp.sim
 
 .. autosummary::
-  :toctree: generated/
+    :toctree: generated/
 
-  sim_cycle
-  sim_synaptic_kernel
+    sim_cycle
+    sim_synaptic_kernel
 
 Combined Signals
 ~~~~~~~~~~~~~~~~
 
-.. currentmodule:: neurodsp.sim.combined
+.. currentmodule:: neurodsp.sim
 
 .. autosummary::
-  :toctree: generated/
+    :toctree: generated/
+
+    sim_combined
+
+Create Data
```

Comment:
I think we should call it something slightly different than `create_data`. It doesn't really create data, but more like meta-data. But actually, I think we could just call these "Utilities" really?

Reply:
Changed to "Utilities"